datasets/dbpedia_14/dbpedia_14.py

```diff
@@ -0,0 +1,146 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The DBpedia dataset for text classification."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@article{lehmann2015dbpedia,
+  title={DBpedia--a large-scale, multilingual knowledge base extracted from Wikipedia},
+  author={Lehmann, Jens and Isele, Robert and Jakob, Max and Jentzsch, Anja and Kontokostas,
+  Dimitris and Mendes, Pablo N and Hellmann, Sebastian and Morsey, Mohamed and Van Kleef,
+  Patrick and Auer, S{\"o}ren and others},
+  journal={Semantic web},
+  volume={6},
+  number={2},
+  pages={167--195},
+  year={2015},
+  publisher={IOS Press}
+}
+"""
+
+_DESCRIPTION = """\
+The DBpedia ontology classification dataset is constructed by picking 14 non-overlapping classes
+from DBpedia 2014. They are listed in classes.txt. From each of thse 14 ontology classes, we
+randomly choose 40,000 training samples and 5,000 testing samples. Therefore, the total size
+of the training dataset is 560,000 and testing dataset 70,000.
+There are 3 columns in the dataset (same for train and test splits), corresponding to class index
+(1 to 14), title and content. The title and content are escaped using double quotes ("), and any
+internal double quote is escaped by 2 double quotes (""). There are no new lines in title or content.
+"""
+
+_HOMEPAGE = "https://wiki.dbpedia.org/develop/datasets"
+
+_LICENSE = "Creative Commons Attribution-ShareAlike 3.0 and the GNU Free Documentation License"
+
+_URLs = {
+    "dbpedia_14": "https://drive.google.com/uc?export=download&id=0Bz8a_Dbh9QhbQ2Vic1kxMmZZQ1k",
```

Comment:
where does the url come from ?

Reply:
I took the same link [here](https://text-docs.readthedocs.io/en/latest/_modules/torchtext/datasets/text_classification.html).